qiita_pet/test/test_study_handlers.py

```diff
@@ -382,5 +382,57 @@ class TestEBISubmitHandler(TestHandlerBase):
     pass
 
 
+class TestDelete(TestHandlerBase):
```

Comment:
I think this tests should include `database = True`; since they can potentially modify the database, in case of malfunction; making a lot of tests to fail and harder to find the original problem.